packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -3873,3 +3993,52 @@ class _CopySelectionAction extends ContextAction<CopySelectionTextIntent> {
   @override
   bool get isActionEnabled => state._value.selection.isValid && !state._value.selection.isCollapsed;
 }
+
+/// A class to return data for the `onContentCommitted` callback on text fields,
+/// which is when image content is inserted into the text field. The class holds
+/// information for the mime type, URI (location), and bytedata for the inserted
+/// content.
+@immutable
+class CommittedContent {
+  /// Creates a [CommittedContent] class. Any parameters can be null.
+  const CommittedContent({this.mimeType, this.uri, this.data});
+
+  /// mime type of inserted content
```

Comment:
Capitalize `m`.  Also, periods at the ends of all the sentences in the docs in this class.